sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/FilesImpl.java

```diff
@@ -594,6 +594,7 @@ Mono<ResponseBase<FilesUploadRangeFromURLHeaders, Void>> uploadRangeFromURL(
                 @HeaderParam("x-ms-file-last-write-time") FileLastWrittenMode fileLastWrittenMode,
                 @HeaderParam("x-ms-allow-trailing-dot") Boolean allowTrailingDot,
                 @HeaderParam("x-ms-source-allow-trailing-dot") Boolean allowSourceTrailingDot,
+                @HeaderParam("x-ms-file-request-intent") ShareTokenIntent fileRequestIntent,
```

Comment:
We are going to need to do a second PR into the STG 92 branch to add x-ms-file-request-intent to File Put Range from URL.  See the email thread "RE: Storage File Put Range from URL missing x-ms-file-request-intent header"